README.md

```diff
@@ -1 +1,62 @@
-# sig-chinese-localization
\ No newline at end of file
+# CDF Chinese Localization Special Interest Group
+
+## Group Charter
+
+### Introduction
+
+There's a very big number of developers who is the Chinese speaker. But English is not the first language of them. People prefer to search and read documents in Chinese. Especially for the regular users.
```

Comment:
First sentence suggestion of this line: There're a very big number of developers who are the Chinese speakers

Reply:
![image](https://user-images.githubusercontent.com/1450685/128603324-8e01186b-7aa9-4815-b914-30408fc89482.png)
![image](https://user-images.githubusercontent.com/1450685/128603336-46da1188-1f18-4666-b806-28225eac3dd6.png)